tests/DataDictionaryTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+use Octri\ConsortReport\DataDictionary,
+    PHPUnit\Framework\TestCase;
+
+/**
+ * @covers DataDictionary
+ */
+final class DataDictionaryTest extends TestCase {
+
+  public function testGetFieldName() {
+    $mockDictionary = array(
+      'screen_id' => array(
+        "field_name" => "screen_id",
+        "form_name" => "screening_id",
+        "field_label" => "Screening Id"
+      ),
+      'dsp_stop_reason' => array(
+        "field_name" => "dsp_stop_reason",
+        "form_name" => "subject_data",
+        "field_label" => "Stop Reason"
+      ),
+      'another_field' => array(
+        "field_name" => "another_field",
+        "form_name" => "another_form",
+        "field_label" => "Another Field"
```

Comment:
Any reason you're using interpolating quotes here?

Reply:
No, my bad. Hard to train myself to use single quotes in PHP. I'll fix it.